cpp/src/slice2cs/Gen.cpp

```diff
@@ -901,7 +901,7 @@ Slice::CsVisitor::getDispatchParams(
     }
 
     string currentParamName = getEscapedParamName(op, "current");
-    params.push_back(getUnqualified("Ice.Current", ns) + " " + currentParamName + " = null");
+    params.push_back(getUnqualified("Ice.Current?", ns) + " " + currentParamName + " = null");
```

Comment:
Kept all this code "as is". 

In a follow-up PR, I want to make this Current parameter non-nullable.